CMakeLists.txt

```diff
@@ -445,3 +445,23 @@ install(
   EXPORT "${PROJECT_NAME}-target"
   NAMESPACE "${PROJECT_NAME}::"
   DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}")
+
+if(BUILD_PACKAGE AND UNIX)
```

Comment:
`UNIX` includes `APPLE` here?

Reply:
Yes, but now I think it should be changed to

``if (BUILD_PACKAGE AND UNIX AND NOT APPLE)``